### ISSUE TITLE
Feat: Custom reference lines for Waveform

### ISSUE THREAD

**Issue by gavalierm** [norihiro/obs-color-monitor#81]
Can you please add 

- padding around the waveform
- reference levels lines for 5% and 95% (or adjustable)

If you have Davinci Resolve the waveform is very user friendly.

**Comment by freeman401** [comment]
Yes please. I would like to have an option to enable additional reference lines for the limited luma range (specifically at values 16 and 235). If this could be done with adjustable lines, it would be great too.
